pkg/multicluster/kubeconfig/kubeconfig_secret_test.go

```diff
@@ -46,6 +46,7 @@ users:
 	})
 
 	Describe("ToSecret", func() {
```

Comment:
nit: let's add new describe blocks for the new functions "ToSecretWithKey" and "SecretToConfigWIthKey"